doc/source/train/user-guides/using-gpus.rst

```diff
@@ -132,7 +145,7 @@ assign each worker a NVIDIA A100 GPU.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 PyTorch Distributed supports multiple `backends <https://pytorch.org/docs/stable/distributed.html#backends>`__
-for communicating tensors across workers. By default Ray Train will use NCCL when ``use_gpu=True`` and Gloo otherwise.
+for communicating tensors across workers. By default Ray Train will use NCCL when ``use_gpu=True`` and Gloo otherwise. For HPU resources, you need to set it to hccl as follows.
```

Comment:
Please add a working example to show case how to use a hccl backend with Ray Train. thx!